src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const checkPartner = person.sex === 'm'
```

Comment:
```suggestion
export const Person = ({ person }) => {
  const { sex, partnerName, name, age, isMarried } = person
  const checkPartner = person.sex === 'm'
```

Reply:
you can destructure an object to not repeat using `person.{field name}` many times :)

Reply:
+ sure